app/src/main/java/com/marvelapp/data/CharacterEntity.kt

```diff
@@ -0,0 +1,6 @@
+package com.marvelapp.data
+
+data class CharacterEntity(
+    val name: String,
+    val image: String? = null,
```

Comment:
1) remove this unnecessary comma
2) it is really necessary that this be nullable and with a default value?

Reply:
Solved